### PR TITLE
Spinner

### PR DESCRIPTION
**Pull request by ghost**
The Spinner is placed in each page that request data to be placed in an array such as the card view. @annamarieHelberg the spinner is placed in a folder "Spinner" in the shared directory for now I will place it there and leave it to you for issue #121. It also uses a shared module so that modules like home component and volumes component can both make use of it simultaneously _for now that was the set up_ so after the change it should not require many changes if any.  

screenshot_2018-07-11 docks-ui

To force view the spinner _as it will be highly unlikely to view in local host_ wrap the ngOnInit() functions in any card view/ list view with the following code snippet:
```javascript
setTimeout(() => {
...//place code
}, 8000)
```

### CODE REVIEW

File: src/app/_shared/services/card-view/services-card-view.component.ts [hunk 1]
```diff
@@ -1,36 +1,54 @@
...
-    }
+          });
+    this.modalObjectTasks = [];
+  }
 
+  public PrettifyDateTime(buff: string): string {
+    return Formatter.PrettifyDateTime(buff);
+  }
 
+<<<<<<< HEAD
```
Comment (egeldenhuys) [inline src/app/_shared/services/card-view/services-card-view.component.ts]: Unmerged changes

**Comment by annamarieHelberg** [comment]
@Paulo-W thanks. As we refactor, we will move it accordingly. I have one question though, why the phrase "gathering honey"?

**Review by annamarieHelberg — DISMISSED** [review]
Can we please change the "gathering honey"? It doesn't make sense

**Comment by ghost** [comment]
screenshot_2018-07-12 docks-ui
Do you prefer this?

**Comment by egeldenhuys** [comment]
How about "Fetching data" ?

**Comment by annamarieHelberg** [comment]
I agree with @egeldenhuys . "Fetching data" is better

**Comment by ghost** [comment]
I will make it fetching data then

File: src/app/_shared/services/list-view/service-list-view.component.ts [hunk 2]
```diff
@@ -1,47 +1,51 @@
...
+
+  public services: Service[] = [];
+  public removeeId = String;
+  public isLoaded = false;
+
+  ngOnInit() {
+        this.serviceService.getServices().subscribe((services) => {
+            this.services = services;
+          });
+          this.isLoaded = true;
```
Comment (egeldenhuys) [inline src/app/_shared/services/list-view/service-list-view.component.ts]: This should be inside the `getServices().subscribe()`

File: src/app/_shared/volumes/list-view/volume-list-view.component.ts [hunk 3]
```diff
@@ -21,6 +23,8 @@ export class VolumeListViewComponent implements OnInit {
             this.volumes = volumes;
             // console.log(volumes);
         }, error => console.error('Error: ' + error));
+
+        this.isLoaded = true;
```
Comment (egeldenhuys) [inline src/app/_shared/volumes/list-view/volume-list-view.component.ts]: This should also only be set once the observable resolves

**Review by egeldenhuys — CHANGES_REQUESTED** [review]
The loading indicators at least provide some comfort

**Review by egeldenhuys — APPROVED** [review]
Spinner now works on all relevant pages :+1:

**Pull request merged**